processor/spanmetricsprocessor/config.go

```diff
@@ -58,6 +59,15 @@ type Config struct {
 	DimensionsCacheSize int `mapstructure:"dimensions_cache_size"`
 
 	AggregationTemporality string `mapstructure:"aggregation_temporality"`
+
+	// skipSanitizeLabel if enabled, labels that start with _ are not sanitized
+	skipSanitizeLabel bool
+}
+
+var dropSanitizationGate = featuregate.Gate{
+	ID:          "exporter.prometheus.PermissiveLabelSanitization",
```

Comment:
I don't think we should have a feature gate that refers to the prometheus exporter on a different component (specially on one that can be used independently of Prometheus)

Reply:
Yes, this should definitely be a separate gate identifier.

Reply:
@mx-psi, @Aneurysm9 updated this. please review.